homeassistant/setup.py

```diff
@@ -516,28 +522,27 @@ async def when_setup() -> None:
 
     listeners: list[CALLBACK_TYPE] = []
 
-    async def _matched_event(event: core.Event) -> None:
+    async def _matched_event(event: EventType[EventComponentLoaded]) -> None:
```

Comment:
I guess we should add EventHassStarted 😄

Reply:
> I guess we should add EventHassStarted 😄

😄 Not sure how useful it would actually be besides knowing that `event.data` is always empty. But that could be helpful sometimes too. Maybe a type alias 🤔 Something along the lines of
```py
EventEmptyData = Mapping[Never, Never]
```